### PR TITLE
Add project git_hook functionality, see:…

### PR DESCRIPTION
**Pull request by liger1978**
Add project git_hook functionality, see: http://docs.gitlab.com/ee/api/projects.html#git-hooks-ee-only

### CODE REVIEW

**Comment by asedge** [comment]
@liger1978, could you please add tests?  Thanks!

**Comment by liger1978** [comment]
OK, tests added.

File: spec/gitlab/client/projects_spec.rb [hunk 1]
```diff
@@ -363,6 +363,85 @@
...
+          with(headers: { 'PRIVATE-TOKEN' => Gitlab.private_token }).
+          to_return(body: '')
+        @git_hook = Gitlab.delete_git_hook(1)
+      end
+
+      it "should get the correct resource" do
+        expect(a_delete("/projects/1/git_hook")).to have_been_made
+      end
+
+    it "should return false" do
```
Comment (asedge) [inline spec/gitlab/client/projects_spec.rb]: Could you fix the indent here? Thanks.

**Comment by liger1978** [comment]
Fixed indentation and squashed the commits into one to keep your commit log clean...

**Comment by liger1978** [comment]
Just to be clear: this should be ready to merge now! Cheers!

**Comment by asedge** [comment]
Thanks!

**Pull request merged**